src/components/progress-bar/index.tsx

```diff
@@ -18,13 +18,15 @@ function ProgressBar({
 	percentDone: number
 	rounded?: boolean
 }) {
+	const portionDone = clamp(percentDone, 0, 100) / 100
+
 	return (
 		<span
 			className={s.root}
 			style={
 				{
 					'--border-radius': rounded ? '2px' : '0px',
-					'--percent-done': `${clamp(percentDone, 0, 100)}%`,
+					'--portion-done': portionDone,
```

Comment:
Switched to `--portion-done` so that the use of `scaleX` is more intuitive in CSS.

Have left the props API the same; could make sense to change in a follow-up PR, but that would require changes to eg. collection progress bars etc, so have left it out of this PR.